dir.props

```diff
@@ -412,4 +412,14 @@
 
   <Import Project="$(RoslynPropsFile)" Condition="'$(OsEnvironment)'=='Unix' and Exists('$(RoslynPropsFile)')" />
 
+  <PropertyGroup>
+    <!-- We don't use any of MSBuild's resolution logic for resolving the framework, so just set these two properties to any folder that exists to skip
+         the GenerateReferenceAssemblyPaths task (not target) and to prevent it from outputting a warning (MSB3644). -->
+    <_TargetFrameworkDirectories>$(MSBuildThisFileDirectory)/Documentation</_TargetFrameworkDirectories>
```

Comment:
I suspect we will want to change this :smile: . I believe this does not work if it is set to the root of corefx (i.e. `$(MSBuildThisFileDirectory)`) because the `bin` directory is underneath that and seemed to be confusing the MSBuild targets as to which files to copy into output directories. I believe `Tools` will also be problematic for the same reason (most of our framework libraries end up living under `Tools` as well).